<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecorderSettings</class>
 <widget class="QDialog" name="RecorderSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>371</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>367</width>
         <height>304</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QWidget" name="widget" native="true">
          <widget class="QWidget" name="">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>321</width>
             <height>261</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Input Device:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QComboBox" name="audioDeviceBox"/>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Audio Codec:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QComboBox" name="audioCodecBox"/>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>File Container:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QComboBox" name="containerBox"/>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Sample rate:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QComboBox" name="sampleRateBox"/>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Channels:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QComboBox" name="channelsBox"/>
            </item>
            <item row="5" column="2">
             <widget class="QComboBox" name="bitrateBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Bitrate:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RecorderSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RecorderSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
